src/ResourceManagement/HDInsight/HDInsight.Tests/Helpers/GetClusterSpecHelpers.cs

```diff
@@ -22,6 +22,8 @@ namespace HDInsight.Tests.Helpers
 {
     public static class GetClusterSpecHelpers
     {
+        private const string ADLClusterRootPath = "/Clusters/SDK";
```

Comment:
You need to add the changes here. This PR will not be accepted without requested changes.